packages/twenty-server/src/engine/core-modules/billing/billing.service.ts

```diff
@@ -154,6 +154,32 @@ export class BillingService {
     return session.url;
   }
 
+  async updateBillingSubscription(user: User): Promise<boolean> {
+    const billingSubscription = await this.getCurrentBillingSubscription({
+      workspaceId: user.defaultWorkspaceId,
+    });
+    const newInterval =
+      billingSubscription?.interval === 'year' ? 'month' : 'year';
+    const billingSubscriptionItem = await this.getBillingSubscriptionItem(
```

Comment:
are all the edge cases covered? what happens if there is no billingSubscription for example?